src/manage-apps/fix-packages.md

```diff
@@ -0,0 +1,68 @@
+# Cleaning Up Packages with Terminal Commands
+
+If your system complains about a failed upgrade, package manager conflicts, broken upgrades, or other package-related issues, there are several common fixes to these problems. Some package manager issues can be resolved with the graphical update program, but many require the command line.
+
+## General Fixes for Apt Packages
+
+If a package was installed with `apt`, these commands perform general cleanup that can resolve many of `apt`'s errors. These commands should be run one at a time.
+
+```bash
+sudo apt clean
+sudo apt update -m
+sudo dpkg --configure -a
+sudo apt install -f
+sudo apt full-upgrade
+sudo apt autoremove --purge
+```
+
+- `apt clean` - The `clean` command clears out the local repository of retrieved package files.
+- `apt update -m` - The `-m` option ignores corrupted package files during a package update.
+- `dpkg --configure -a` - The `--configure -a` command configures any unpacked but not yet configured packages.
+- `apt install -f` - The `-f` option attempts to correct broken dependencies
+- `apt full-upgrade` - In addition to downloading and installing package updates, `full-upgrade` downgrades or removes dependencies as necessary when upgrading packages.
+- `apt autoremove --purge [packagename]` - In addition to removing a package, `autoremove` will remove dependencies no longer required by any installed application. Combining with `purge` will remove any residual configuration files related to the package.
+- `apt autoremove --purge` - Running this command with no specified package will remove any packages that were previously dependencies for other installed packages but are no longer required (either because the dependent package was removed, or because the package was updated to no longer depend on certain packages.)
+
+## Fixing Individual Apt Packages
+
+### The Reinstall Option
+
+You may see packages that are still broken and need to be installed manually or purged manually. This may indicate that the package has broken or cyclical dependencies.
+
+This command reinstalls the package, which can be useful if the package has many reverse dependencies (the packages that depend on a given package):
+
+```bash
+sudo apt install --reinstall [packagename]
+```
+
+### The Purge Option
+
+>**Note**: Be careful when using `purge` and `autoremove`. Verify the terminal output to confirm the command will only affect the packages you are trying to fix. If unrecognized packages are removed causing unexpected changes, run `sudo apt install pop-desktop` and reboot to ensure those critical Pop!\_OS components are reinstalled.
+
+This command will remove a package and its system-wide configuration files. Use it to remove a package that is causing issues:
+
+```bash
+sudo apt purge [packagename]
+```
+
```

Comment:
I think explaining the usage and output of `apt policy [packagename]` could be a valuable addition. That command shows the available versions of a package, and sometimes it's necessary to do that in order to tell apt to install a specific version (with `sudo apt install packagename=version`).

Reply:
Added